api/src/main/java/org/openmrs/module/locationbasedaccess/common/Constants.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.locationbasedaccess.common;
+
+public class Constants {
+
+    public static final String ACCESS_LOCATION_PERSON_ATTRIBUTE_TYPE_NAME = "accessLocation";
+
+    public static final String PATIENT_OBJ_NAME = "PATIENT";
+
+    public static final String SAVE_PATIENT_METHOD_NAME = "savePatient";
```

Comment:
This need to be updated as `saveUser` to make those changes.

Reply:
We have not yet got to the ticket which deals with users.